lib/widget/download/download_grid.dart

```diff
@@ -55,6 +55,8 @@ class _DownloadGridState extends State<DownloadGrid> {
                 height: resolveIconSize(fileType),
                 child: SvgPicture.asset(
                   FileUtil.resolveFileTypeIconPath(fileType.name),
+                  colorFilter:
+                      ColorFilter.mode(Colors.white60, BlendMode.srcIn),
```

Comment:
Using a white color for all types of files makes the UI look soulless and bland in my opinion. I would like to have those distinct colors back for each file type.

Reply:
i fix this issue